src/fides/ctl/core/config/utils.py

```diff
@@ -40,6 +41,53 @@ def get_config_from_file(
     return None
 
 
+def check_if_required_config_vars_are_configured() -> None:
+    app_encryption_key: Union[str, int, None] = getenv(
+        "FIDES__SECURITY__APP_ENCRYPTION_KEY"
+    )
+    oauth_root_client_id: Union[str, int, None] = getenv(
+        "FIDES__SECURITY__OAUTH_ROOT_CLIENT_ID"
+    )
+    oauth_root_client_secret: Union[str, int, None] = getenv(
+        "FIDES__SECURITY__OAUTH_ROOT_CLIENT_SECRET"
+    )
+    try:
+        app_encryption_key = get_config_from_file("", "security", "app_encryption_key")
+        oauth_root_client_id = get_config_from_file(
+            "", "security", "oauth_root_client_id"
+        )
+        oauth_root_client_secret = get_config_from_file(
+            "", "security", "oauth_root_client_secret"
+        )
```

Comment:
These will run any time there is a config file and could lead to false positives since some values could be set in the environment variables and others in the toml file. Environment variables are supposed to take precedence here so you could only check the toml file if the variable is `None` after checking the env var.

```py
try:
    if not app_encryption_key:
        app_encryption_key = get_config_from_file("", "security", "app_encryption_key")
    ...
```

Reply:
Good catch. Updated!